Samples/UIWidgetsSamples_2019_4/Assets/NativeConsole.cs

```diff
@@ -0,0 +1,21 @@
+﻿using UnityEngine;
```

Comment:
move this file inside com.unity.uiwidgets/Runtime/engine2/utils/ and rename it as native_console.cs.